docs/spark-standalone.md

```diff
@@ -328,6 +328,17 @@ SPARK_WORKER_OPTS supports the following system properties:
   </td>
   <td>3.0.0</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.service.db.backend</code></td>
+  <td>LEVELDB</td>
+  <td>
+    When <code>spark.shuffle.service.db.enabled</code> is true, user can use this to specify the kind of disk-based 
+    store used in shuffle state store. This supports `LEVELDB` and `ROCKSDB` now and `LEVELDB` as default value. 
+    This only affects standalone mode (yarn always has this behavior enabled). 
+    The original data store in `LevelDB/RocksDB` will not be automatically convert to another kind of storage now.
```

Comment:
Do you mean the data that is stored in LevelDB is not available after changing to RocksDB for example?

Reply:
Yes, there is no function to automatically switch `*.ldb` to `*.rdb` now.